code/ship/ship.cpp

```diff
@@ -2728,7 +2728,7 @@ int parse_warp_params(const WarpParams *inherit_from, WarpDirection direction, c
 		if (accel_exp >= 0.0f)
 			params.accel_exp = accel_exp;
 		else
-			Warning(LOCATION, "%s specified as less than 0 on %s '%s'; value ignored", str, info_type_name, info_name);
+			error_display(0, "%s specified as less than 0 on %s '%s'; value ignored", str, info_type_name, info_name);
```

Comment:
Is there a reason to prefer `error_display`? I'll admit that `Warning(..` is much more intuitive to me than `error_display(0, ..`, I can never remember which one 0 or 1 is.

Reply:
Increasing number = increasing severity, so 0 = warning and 1 = error.

The `error_display` function is specifically intended to be used while parsing.  It displays the file name and line number of the file being parsed; and it also maintains warning and error counts for parsing, distinct from "all" warning and error counts.

Regular `Warning` and `Error` do not display anything about the file being parsed.